### ISSUE TITLE
Memory spike on startup

### ISSUE THREAD

**Issue by nikolayemrikh** [oraoto/pib#31]
Is there any solution to lower memory spike on startup? Currently it spikes up to 3GB, then lowers back to 300MB a few seconds later, but this spike can crush page on weak devices

**Comment by oraoto** [comment]
Hello, what's your browser?

Currenty, Chromium (76) consumes  a lot of memory to optimize WASM, and you can verify this using:

``` 
chromium --js-flags="--no-wasm-tier-up"
```

**Comment by nikolayemrikh** [comment]
Yes, with this flag there're no memory spike.. Is there any way to resolve this problem without flags? Maybe compile with some other options may help?

> Hello, what's your browser?

Literally every browser (FF, Safari, Chromiums)
